build.sh

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin
+packages="contracts common-types machine cf.js node node-provider playground dapp-high-roller"
+
+for package in $packages; do
+  echo ">>> Building package: $package"
+  cd packages/$package
+  yarn build
+  cd ../..
```

Comment:
Protip: you can also do `cd -` here to jump back

Reply:
Good one!